token-lending/program/src/processor.rs

```diff
@@ -785,6 +785,7 @@ fn process_redeem_reserve_collateral(
     let clock = &Clock::get()?;
     let token_program_id = next_account_info(account_info_iter)?;
 
+    _refresh_reserve_interest(program_id, reserve_info, clock)?;
```

Comment:
This was originally here then i removed it when implementing outflow limits
https://github.com/solendprotocol/solana-program-library/pull/125/files#diff-724dc1de5d8ee8e4daf8d2a6d0c0c12bde2a03921e3cc74bcb75ab0caf3d00f1L637